infoschema/infoschema.go

```diff
@@ -300,6 +308,8 @@ func (h *Handle) EmptyClone() *Handle {
 
 // Schema error codes.
 const (
+	codeTableLocked terror.ErrCode = 1
```

Comment:
It's not a good idea to use the error code `1`

Reply:
em... Any advice? Because MySQL doesn't have this error.

Reply:
I think it's fine to add it as a self-defined TiDB error: https://github.com/pingcap/parser/blob/89ae120307cc42a1cccc4daba2c589966db77055/mysql/errcode.go#L936

@winkyao @zimulala

Reply:
yep, agree with @bb7133